src/main/java/org/folio/circulation/CirculationVerticle.java

```diff
@@ -42,7 +43,8 @@ public void start(Future<Void> startFuture) {
 
     this.server = vertx.createHttpServer();
 
-    new CheckOutByBarcodeResource(client).register(router);
+    new RegularCheckOutByBarcodeResource(client).register(router);
```

Comment:
Why has this been renamed to include `Regular` in the name?

I imagine it is to try to better distinguish between a check out and an overridden check out. What confuses me is that we didn't use this naming for overridden renewals. If we are going to use this convention, I think we need to be consistent.

I personally don't find that `Regular` adds any extra information.

Reply:
Refactored _CheckOutByBarcodeResource_ to replace inheritance with aggregation.